src/main/java/org/humancellatlas/ingest/project/ProjectService.java

```diff
@@ -73,6 +73,20 @@ public Project register(final Project project) {
         return persistentProject;
     }
 
+    public Project createSuggestedProject(final ObjectNode suggestion) {
+        Project suggestedProject = new Project(null);
+        suggestedProject.setWranglingState(WranglingState.NEW_SUGGESTION);
+        var notes = String.format(
+            "DOI: %s \nName: %s \nEmail: %s \nComments: %s",
```

Comment:
Why are we adding spaces after the string parameters? Are those spaces needed there?

Reply:
It's easier to read like this. Not necessary but not harmful and makes the code easier on the eye